test/support/test_helpers.ex

```diff
@@ -1,9 +1,25 @@
 defmodule CodeCorps.TestHelpers do
   use Phoenix.ConnTest
+  import ExUnit.Assertions
 
   def ids_from_response(response) do
     Enum.map response["data"], fn(attributes) ->
       String.to_integer(attributes["id"])
     end
   end
+
+  def assert_jsonapi_relationship(json = %{"relationships" => relationships}, relationship_name, id) do
+    assert relationships[relationship_name]["data"]["id"] == Integer.to_string(id)
+    json
+  end
+
+  def assert_jsonapi_relationship(json, relationship_name, id) do
+    assert json["data"]["relationships"][relationship_name]["data"]["id"] == Integer.to_string(id)
```

Comment:
Remind me, will this fail if the json is completely invalid, as in, for example, no "data" key at all?

Maybe 3 signaturs would do well. One for just the "data" part, one for the entire payload (this one, but specify the structure, like in the previous method, and one which is just

```Elixir
def assert_jsonapi_relationship(_, _, _), do: false
```